rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/core/RestExceptionHandler.java

```diff
@@ -43,8 +45,8 @@
         return new ResponseEntity<>(new ErrorMessage(e, HttpStatus.FORBIDDEN), HttpStatus.FORBIDDEN);
     }
 
-    @ExceptionHandler(ResourceNotFoundException.class)
-    public ResponseEntity<ErrorMessage> handleResourceNotFound(ResourceNotFoundException e) {
+    @ExceptionHandler({ResourceNotFoundException.class, WrongPaginationParameterException.class})
```

Comment:
Hmm I am unhappy to send 404 in this case.
If we want to send a Client Error we should use 406/409 otherwise we should use 500.

Reply:
What do you think about `400 Bad Request` ?